.github/workflows/appsignals-e2e-ec2-test.yml

```diff
@@ -71,10 +71,10 @@ jobs:
         working-directory: testing/terraform/ec2
         run: |
           old_command='"wget -O adot.jar ${var.adot_jar}",'
-          new_command='"aws s3 cp s3://main-build-adot-jar/aws-opentelemetry-agent.jar ./adot.jar",'
+          new_command='"aws s3 cp s3://main-build-adot-staging-jar/aws-opentelemetry-agent.jar ./adot.jar",'
           sed -i "s|$old_command|$new_command|" "main.tf"
           
-          if grep -Fq '"aws s3 cp s3://main-build-adot-jar/aws-opentelemetry-agent.jar ./adot.jar",' main.tf; then
+          if grep -Fq '"aws s3 cp s3://main-build-adot-staging-jar/aws-opentelemetry-agent.jar ./adot.jar",' main.tf; then
```

Comment:
Is just making changes within a terraform file? Can a terraform variable be used instead?

Reply:
Do you mean input as a terraform variable for `s3://main-build-adot-staging-jar/aws-opentelemetry-agent.jar`?
I'm replacing the entire line because this piece of code is for only testing staging images. I could put `s3://main-build-adot-staging-jar/aws-opentelemetry-agent.jar` as a github env variable instead?

Reply:
Use a variable [here](https://github.com/aws-observability/aws-otel-java-instrumentation/blob/0ffa2a74724caf8467284306af25a69b37715e79/testing/terraform/ec2/main.tf#L105). Set the variable to be the command you want to run. I don't think you need to use command line here.

Reply:
I think that could work

Reply:
Done